### ISSUE TITLE
[Crash]: Extra utilities 2 and Applied energetics 2 issues?

### ISSUE THREAD

**Issue by Nukinunu** [FTBTeam/FTB-University#296]
* What version of University: 1.2.1
* Have you added any mods: No
* If yes, which mods:
* Crash report link (preferably Pastebin etc, not the file): https://pastebin.com/01CxUucd
* What were you doing when the crash happened: i walking back to my Base, the chunks were loading
* Any other details: i was hearing glass breaking sounds, and then i noticed some of the M.E fluix cables dissapeared, it still was there but Invisible,

**Comment by aaronhowser1** [comment]
Does it crash when you try rejoining?

**Comment by Nukinunu** [comment]
yeah but then i rejoined again after that crash and i could play some, some hours later the minecraft crashed by the same error with those 2 mods, rejoined and crashed with the same 2 mods, joined again and i was able to play
